pkgs/by-name/st/still/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  meson,
+  ninja,
+  pkg-config,
+  pixman,
+  kdePackages,
+  wayland-scanner,
+  scdoc,
+}:
+
+let
+  version = "0.0.7";
+in
+stdenv.mkDerivation {
+  pname = "still";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "faergeek";
+    repo = "still";
+    tag = "v${version}";
+    hash = "sha256-tPDPEUBVfNNnTULRNuqyshfvjb1otiko3KlsAj46qRY=";
+  };
+
+  buildInputs = [
+    pixman
+    kdePackages.wayland
+    kdePackages.wayland-protocols
```

Comment:
`wayland` and `wayland-protocols` are available in the toplevel scope, is there a reason we're getting those from `kdePackage`?